src/test/java/core/basesyntax/DataParserTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.impl.DataParserImpl;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class DataParserTest {
+    private DataParser dataParser;
+
+    @BeforeEach
+    public void setUp() {
+        dataParser = new DataParserImpl();
+    }
+
+    @Test
+    public void parseData_ValidData_Ok() {

```

Comment:
We need to use lowerCamelCase in such cases. Review the project and ensure that the correct naming is being used.
parseData_validData_ok`

Reply:
not fixed 
fix it in all test names